setup.py

```diff
@@ -5,8 +5,12 @@
 
 """
 from setuptools import setup, find_packages
+import sys
 
 
+if sys.version_info[0] == 3 and sys.version_info[1] <= 5:
```

Comment:
If `sys.version_info[0]` is `2`, then it won't generate the error. You can also use this simple construct: `sys.version_info < (3, 6)`.

Other nice examples:
```
 ~ ➜ python2 -c "import sys; print(sys.version_info<=(3, 6), sys.version_info)"
(True, sys.version_info(major=2, minor=7, micro=14, releaselevel='final', serial=0))

 ~ ➜ python3 -c "import sys; print(sys.version_info<=(3, 6), sys.version_info)"
False sys.version_info(major=3, minor=6, micro=3, releaselevel='final', serial=0)
```